pkg/secrethub/secret_version.go

```diff
@@ -21,6 +22,7 @@ var (
 	ErrSecretTooBig         = errClient.Code("secret_too_big").Error(fmt.Sprintf("maximum size of a secret is %s", units.BytesSize(MaxSecretSize)))
 	ErrEmptySecret          = errClient.Code("empty_secret").Error("secret is empty")
 	ErrCannotWriteToVersion = errClient.Code("cannot_write_version").Error("cannot (over)write a specific secret version, they are append only")
+	ErrSecretNotFound       = errClient.Code("secret_not_found").StatusErrorPref("cannot find secret: %s, error: %v", http.StatusNotFound)
```

Comment:
Usually we concatenate errors without saying `error: ` again, so that we don't repeat ourselves. The error can simply be: `cannot find secret "path/to/secret": problem why the secret cannot be found`